src/Network/Network/LoadBalancer/SetAzureLoadBalancerBackendPool.cs

```diff
@@ -0,0 +1,211 @@
+
+            // Confirm if resource already exists
+            try
+            {
+                existingloadBalancerBackendAddressPool = this.NetworkClient.NetworkManagementClient.LoadBalancerBackendAddressPools.Get(this.ResourceGroupName, this.LoadBalancerName, this.Name);
+            }
+            catch (Rest.Azure.CloudException exception)
+            {
+                if (exception.Response.StatusCode == System.Net.HttpStatusCode.NotFound)
+                {
+                    existingloadBalancerBackendAddressPool = null;
+                }
+                else
+                {
+                    throw;
+                }
+            }
+
+            // Return error message to user if loadbalancer pool doesnt exist
+            if (existingloadBalancerBackendAddressPool == null)
+            {
+                throw new Exception(string.Format("A BackendPool with name '{0}' in resource group '{1}' not found under LoadBalancer '{2}'. Please use New-AzLoadBalancerBackendAddressPool to create a BackendPool .",
+                    this.Name, this.ResourceGroupName, this.LoadBalancerName));
+            }
+
+            // add loadbalancer address to the request available. 
+            if (this.IsParameterBound(c => c.LoadBalancerBackendAddress))
+            {
+                backendAddressPool = SetupBackendPoolWithLoadBalancerAddresses();
+            }
+
+            ConfirmAction(
+                Force.IsPresent,
+                string.Format("Updated Backendpool '{0}' in resource group '{1}' under loadbalancer '{2}'.", this.Name, this.ResourceGroupName, this.LoadBalancerName),
+                Properties.Resources.UpdatingLongRunningOperationMessage,
+                Name,
+                () =>
+                {
+                    var loadBalancerBackendAddressPool = this.NetworkClient.NetworkManagementClient.LoadBalancerBackendAddressPools.CreateOrUpdate(
+                        this.ResourceGroupName, this.LoadBalancerName, this.Name, backendAddressPool);
+
+                    var loadBalancerBackendAddressPoolModel = NetworkResourceManagerProfile.Mapper.Map<PSBackendAddressPool>(loadBalancerBackendAddressPool);
+
+                    WriteObject(loadBalancerBackendAddressPoolModel);
+                });
+        }
+
+        private BackendAddressPool SetupBackendPoolWithLoadBalancerAddresses()
+        {
+            var backendAddressPool = new BackendAddressPool();
+
+            backendAddressPool.LoadBalancerBackendAddresses = new List<LoadBalancerBackendAddress>();
+
+            foreach (var psBackendAddress in this.LoadBalancerBackendAddress)
+            {
+                var backendAddress = ToLoadBalancerBackendAddress(psBackendAddress);
+                backendAddressPool.LoadBalancerBackendAddresses.Add(backendAddress);
+            }
+
+            return backendAddressPool;
+        }
+
+        private BackendAddressPool SetupBackendPoolWithInputObject()
```

Comment:
I think that we probably want to support only this

Reply:
This is due to feedback from the powershell team, see this. https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/583